dandiapi/api/views/zarr.py

```diff
@@ -204,6 +204,7 @@ def delete_files(self, request, zarr_id):
             pattern=r'.*',
         )
     ],
+    operation_id='zarr_file',
```

Comment:
Based on the difference in paths between the two clashing endpoints, I thought a name like `zarr_path_read` would be good, but I think something more descriptive than `zarr_file` is warranted.

Reply:
Renamed to `zarr_content_read`